crate/content.md

```diff
@@ -1,6 +1,8 @@
 # What is Crate?
 
-Crate is an Elastic SQL Data Store. Distributed by design, Crate makes centralized database servers obsolete. Realtime non-blocking SQL engine with full blown search. Highly available, massively scalable yet simple to use.
+Crate is a scalable SQL database with the NoSQL goodies.
+Crate allows to query and compute data with SQL in real time by providing a distributed aggregation engine, native search and super simple scalability.
+It offers auto-sharding and replication, super-fast multi index queries, distributed aggregations and sort, superfast full-text search, and super simple cluster management.
```

Comment:
These three lines should just be one line according `markdownfmt`. Which is why the test is failing :wink:.